modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/EmptyState.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import {ClayDropDownWithItems} from '@clayui/drop-down';
+import ClayEmptyState from '@clayui/empty-state';
+import React from 'react';
+
+import './../css/all_section.scss';
+
+interface IEmptyStateProps {
+	description?: string;
+	title?: string;
+}
+
+const EmptyState: React.FC<IEmptyStateProps> = ({
+	description = Liferay.Language.get(
+		'click-new-to-create-your-first-content'
+	),
+	title = Liferay.Language.get('no-assets-yet'),
```

Comment:
Do we need the `description` and `title` as a prop? Maybe we don't need to change it at all

Reply:
Yes, because at least Content section has different title/description